src/foraging_gui/Dialogs.py

```diff
@@ -9,13 +9,16 @@
 
 import numpy as np
 from matplotlib.backends.backend_qt5agg import NavigationToolbar2QT as NavigationToolbar
-from PyQt5.QtWidgets import QApplication, QDialog, QVBoxLayout
+from PyQt5.QtWidgets import QApplication, QDialog, QVBoxLayout, QHBoxLayout
 from PyQt5 import QtWidgets, uic
-from PyQt5.QtCore import QThreadPool,Qt
+from PyQt5.QtCore import QThreadPool,Qt, QAbstractTableModel
+from PyQt5.QtSvg import QSvgWidget
 
 from MyFunctions import Worker
 from Visualization import PlotWaterCalibration
+from aind_auto_train.curriculum_manager import CurriculumManager
 
+logger = logging.getLogger(__name__)
```

Comment:
What is this line doing?

Reply:
I meant to add a logging info but I forgot. Now I added it at line 1752.

Reply:
The thing that jumped out at me is that you are calling the logger in a different way from how I am. I just use `logging.info(<string>)`. I dont know what logging.getLogger(__name__) does. 

I'll approve the PR, because this is a minor detail. but we should be consistent

Reply:
oh, I think it's better to define a logger for each module by
```python
logger = logging.getLogger(<module_name>)
logger.info('...')
```
see here https://www.codingexplorations.com/blog/best-practices-for-logging-in-python-a-comprehensive-guide

Reply:
That site says that making a logger for each module helps customize the format, and track how messages are generated. Do you know if each module's logger inherits the format of the root logger?

I just made the root logger automatically log the module and line number

Reply:
Yes, once `logging.basicConfig` is called, all `logging.getLogger` will return logger object that inherits the root logger, i.e., streaming everything to the log file specified in basicConfig. For example, here is a log file I got:

![image](https://github.com/AllenNeuralDynamics/dynamic-foraging-task/assets/24734299/cedbef6d-247c-4712-ba0c-c7c4d5d997b1)

The highlighted logs were from my modules with `logger = logging.getLogger`.

Reply:
btw, we can add `logging.StreamHandler()` to `handlers` in `basicConfig` to also send a copy of log info to the console (instead of `print` @XX-Yin)
```python
logging.basicConfig(
    format=log_format,
    level=logging.INFO,
    datefmt=log_datefmt,
    handlers=[logging.FileHandler(logging_filename),
              logging.StreamHandler()]
    )
```

Reply:
awesome!